CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 - Changed: `comment-word-blacklist` no longer ignores words within copyright  comments
 - Changed: `font-family-name-quotes` now checks the `font` property in addition to the `font-family` property
 - Changed: `unit-*` rules now check `@media` values too.
+- Changed: better handling of vendor prefixes in `keyframe-declaration-no-important`, `no-unknown-animations`, `time-no-imperceptible`.
```

Comment:
Can we break this into three:

- Changed: `keyframe-declaration-no-important` now checks vendor prefixed `@keyframes` at-rules.
- Changed:  `time-no-imperceptible` now checks vendor prefixed properties.
- Fixed: `no-unknown-animations` now classifies vendor prefixed `@keyframes` at-rules are known.